### DISCUSSION TITLE
Update Google Play Store version regularly

### DISCUSSION THREAD

**Discussion by NomarCub** [iSoron/uhabits#964] [Feature Requests]
I thought 2.0 was still in alpha up until now, as the Play Store has 1.8.

**Comment by iSoron** [comment]
Hi @NomarCub, Loop 2.0 is currently in beta. You can get it by [joining our open beta on Google Play](https://play.google.com/apps/testing/org.isoron.uhabits) or downloading it from [F-Droid](https://f-droid.org/).

**Reply by NomarCub** [reply]
Thanks!
The naming is confusing then. The alpha github release is marked as alpha, but nothing suggested 2.0.2 was a beta on F-Droid or github.

**Comment by arxeiss** [comment]
I see that in Github is already v2.1.1 released this September, but in Google Play is still version 2.0.3 from last year. Do you plan to update it there too? Thanks

**Reply by cb3inco** [reply]
@iSoron is there any plans to update Google Play app?

**Comment by liwsoft** [comment]
Is there any roadmap of updates? For example, by when the beta is over.

**Reply by vatras3** [reply]
@iSoron any news on this?
